wpilibj/src/test/java/edu/wpi/first/wpilibj/geometry/Translation2dTest.java

```diff
@@ -86,4 +86,15 @@ void testDistance() {
     Translation2d two = new Translation2d(6, 6);
     assertEquals(one.getDistance(two), 5 * Math.sqrt(2), kEpsilon);
   }
+
+  @Test
+  void testUnaryMinus() {
+    Translation2d original = new Translation2d(-4.5, 7);
+    var inverted = original.unaryMinus();
```

Comment:
why `var` here?

Reply:
Because it's obvious what the type is and `var` is less typing. it's like `auto` in C++.

Reply:
but why not var for original too?

Reply:
I dunno. No reason not to.

Reply:
In general, I used the full typename when explicitly allocating a new object and then used `var` if I'm calling a method whose return type is obvious. If it is desired for me to be more consistent with either not using `var` at all or using it more often, please let me know.

Reply:
Use `var` just as you would with C++'s `auto`. You already specified the type is Translation2d on the right-hand side, so there's no need to repeat yourself.

Reply:
I will change some occurrences to use `var` more to make it more consistent with how I've been using `auto` in the C++ tests.